components/Intro/TextBox.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+const TextBox = ({ title, text, inView }: any) => {
+  const isTennis = title === "tennis";
+
+  return (
+    <div
+      className={`md:max-w-[22rem] self-center md:justify-self-start ${
```

Comment:
should remove `md:justify-self-start`

Reply:
“justify” is set for special screen size